cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -705,11 +738,15 @@ notifyBlockForging fStats tr = Tracer $ \case
             (\fts -> (fts { ftsNodeIsLeaderNum = ftsNodeIsLeaderNum fts + 1
                           , ftsLastSlot = slot },
                       ftsNodeIsLeaderNum fts + 1))
-  Consensus.TraceForgedBlock {} ->
+  Consensus.TraceForgedBlock {} -> do
     traceCounter "blocksForgedNum" tr
       =<< mapForgingCurrentThreadStats fStats
             (\fts -> (fts { ftsBlocksForgedNum = ftsBlocksForgedNum fts + 1 },
                        ftsBlocksForgedNum fts + 1))
+    meta <- mkLOMeta Critical Public
+    let tBlocksUncoupled = fsBlocksUncoupled fStats
+    traceI tr meta "mySelectedBlocks" =<< STM.modifyReadTVarIO tBlocksUncoupled (+ 1)
```

Comment:
Yep, it's a `+1` because `blocksUncoupledDelta` counts blocks not on the current chain. And even though we just forged it, it's not on the chain until we get a corresponding `AddedToCurrentChain, which is handled higher up in the diff. 👍